forslag.md

```diff
@@ -2,67 +2,62 @@
 
 ***Folketinget pålægger den offentlige administration i højest mulig grad at licensere og udgive offentlige IT-udviklingsopgaver åbent. I tilfælde hvor et system ikke kan offentliggøres af sikkerhedsmæssige årsager, skal staten sikre ejerskab af både kode, data og infrastruktur.
 Koden bag offentlige IT-systemer skal med andre ord være tilgængelig for såvel borger som stat. Det vil sikre billigere systemer, højere kvalitet og dermed vækst.***
+<!-- Det får det til at lyde som om at vækst er formålet med forslaget, så måske skulle vi omformulere denne? -->
 
 ## Introduktion
 Offentlige IT-projekter udgør en væsentlig del af den infrastruktur, der får Danmark til at fungere.
-Når IT-systemer ikke fungerer, er det i sidste ende borgerne, der betaler &ndash; gennem skattebilletten, eller ved at spilde tid på systemer, der ikke fungerer.
+Når IT-systemer ikke fungerer, er det i sidste ende borgerne, der betaler &ndash; gennem skatten, eller ved at spilde tid på systemer, der ikke fungerer.
 Den offentlige sektor har gang på gang udviklet projekter, der er gået over budget, eller som har slået fuldstændig fejl. I værste fald er de aldrig  blevet taget i brug <sub>[citation needed]</sub>
 Mange af disse problemer kunne være undgået<sup><a name="itu">1</a></sup>.
 
 Med dette forslag sikrer vi åbenhed og rimelighed i vores livsvigtige digitale infrastruktur.
-Lige nu er store dele af vores digitale liv ejet af få private virksomheder. Derfor er vi som land dybt afhængige af dem &ndash; og det er dyrt og inffektivt!
+Lige nu er store dele af vores digitale liv ejet af få private virksomheder. Derfor er vi som land dybt afhængige af dem &ndash; og det er dyrt og ineffektivt!
 Lad os sammen skabe en fremtid hvor Danmark går forrest med billig, moderne og innovativ IT.
 
 ### Offentlig IT er billig IT
 Når staten køber et IT system i dag køber de katten i sækken.
 Staten ejer ikke engang systemet.
 Det gør den virksomhed, der har bygget det.
-Tænk hvis det også gjaldt for din bil: så ville producenten svejse motorhjelmen fast så snart den blev solgt.
-Selv ikke din mekaniker måtte få lov til at kigge i den.
-Så hvis bilen gik i stykker, ville du være tvunget til at køre den ødelagte bil hjem Asien, hvor den blev bygget.
+Tænk hvis det også gjaldt for din bil: Så ville producenten gøre teknikken i bilen utilgængelig for selv din mekaniker, og du ville derfor være helt afgængig af producenten til at reparere bilen igen.
```

Comment:
suggestion: "til at reparere bilen igen" -> "for at få bilen repareret"

Reply:
God suggestion, hvordan går man så ind og tilføjer det?

Reply:
Man laver en ændring på branchen:

- På forsiden for repositoriet (https://github.com/offentligit/borgerforslag) er der en dropdown med branches. Vælg `kasperranum-forslag-update` (som er navnet på denne branch). Det tager dig til https://github.com/offentligit/borgerforslag/tree/kasperranum-forslag-update
- Find filen og ret i den ligesom du gjorde før.
- Når du er færdig skal du nederst vælge "Commit directly to the `kasperranum-forslag-update` branch." Så laver den ændringen så den kommer med i dette Pull Request.

Sig til hvis det er forvirrende, så kan jeg godt gøre det, men tænker det også er godt for dig at prøve :smile:

Reply:
Jeg må indrømme at jeg bedre kan lide den meget visuelle version med tilsvejset motorhjelm. "Teknikken" er mere abstrakt.